exec/java-exec/src/main/java/org/apache/drill/exec/util/MemoryAllocationUtilities.java

```diff
@@ -58,7 +58,15 @@ public static void setupBufferedOpsMemoryAllocations(final PhysicalPlan plan, fi
     // if there are any sorts, compute the maximum allocation, and set it on them
     if (bufferedOpList.size() > 0) {
       final OptionManager optionManager = queryContext.getOptions();
-      final long maxWidthPerNode = optionManager.getOption(ExecConstants.MAX_WIDTH_PER_NODE_KEY).num_val;
+      final long maxWidthPerNode;
+      long maxWidth = optionManager.getOption(ExecConstants.MAX_WIDTH_PER_NODE);
```

Comment:
This code appears twice. Can we move it into a function so it appears once? Maybe on to a purpose-build Validator class?